srtcore/common.cpp

```diff
@@ -196,7 +196,7 @@ void CTimer::sleepto(uint64_t nexttime)
 
    while (t < m_ullSchedTime)
    {
-#ifndef NO_BUSY_WAITING
+#if defined(USE_BUSY_WAITING) && USE_BUSY_WAITING
```

Comment:
AFAIR it should work if you simply say `#if USE_BUSY_WAITING`. I remember there was a problem with that somewhere, but then you can simply make sure it's defined in the build file.

Reply:
You are right. I had issue when I needed a value,  like `#if (A == 0)` and `A` was not defined. In case of `USE_BUSY_WAITING` and `ENABLE_IST_DEVIATION`, a simple `#if` will do the job.